README.md

```diff
@@ -8,18 +8,25 @@ The automatically generated website can be viewed at https://iroboteducation.git
 Install required dependencies:
 
 ```bash
+sudo apt-get update
+sudo apt-get -y install python3-pip
+sudo apt-get -y install mkdocs
 pip3 install mkdocs
 pip3 install mkdocs-material
 ```
 
 Build the documentation locally:
 
 ```bash
-mkdocs serve
+python3 -m mkdocs serve
```

Comment:
Something seems not right here -- if you install mkdocs with apt, you shouldn't need to invoke it in this way,

Reply:
Reverted this change in commit 8259333